packages/invoices/src/OutboundShipment/DetailView/types.ts

```diff
@@ -29,4 +30,8 @@ export type CustomerInvoiceAction =
       type: ActionType.SortBy;
       payload: { column: Column<ItemRow> };
     }
-  | CustomerInvoiceUpdateInvoice;
+  | CustomerInvoiceUpdateInvoice
+  | {
+      type: ActionType.AddLine;
+      payload: InvoiceLine;
+    };
```

Comment:
Nit: pretty redundant but the convention is generally to use `payload: { invoiceLine }` heh